<ui version="4.0" >
 <class>SelectUpdatesWidget</class>
 <widget class="QWidget" name="SelectUpdatesWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>233</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>12</number>
   </property>
   <property name="leftMargin" >
    <number>16</number>
   </property>
   <property name="topMargin" >
    <number>16</number>
   </property>
   <property name="rightMargin" >
    <number>16</number>
   </property>
   <property name="bottomMargin" >
    <number>16</number>
   </property>
   <item>
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>There are new versions available of Last.fm or some of its plugins.</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="updatesList" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Please select the ones you want to update and click Next to have them downloaded and installed automatically.</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
